plugins/in_storage_backlog/sb.c

```diff
@@ -110,6 +111,8 @@ static int cb_queue_chunks(struct flb_input_instance *in,
         mk_list_del(&sbc->_head);
         flb_free(sbc);
 
+	flb_input_chunk_update_output_instances(ic, flb_input_chunk_get_size(ic));
```

Comment:
nit: formatting?

Reply:
updated. Move the implementation to https://github.com/fluent/fluent-bit/blob/master/src/flb_input_chunk.c#L307